instance.tf

```
@@ -0,0 +1,75 @@
+resource "aws_instance" "public_instance" {
+  ami                         = "ami-06e46074ae430fba6"
+  associate_public_ip_address = true
+  instance_type               = "t2.micro"
+  key_name                    = "project_key"
+  vpc_security_group_ids      = [aws_security_group.pub_sec_group.id]
+  subnet_id                   = aws_subnet.public[1].id
+  tags = {
+    Name = "${var.env_prefix}-pub_instance"
+  }
+}
+
+resource "aws_security_group" "pub_sec_group" {
```

Do not repeat resource type in resource name (not partially, nor completely):
```suggestion
resource "aws_security_group" "public" {
```